src/declarations/stencil-public-compiler.ts

```diff
@@ -917,7 +917,7 @@ export interface CompilerSystem {
   /**
    * Add a callback which will be ran when destroy() is called.
    */
-  addDestory(cb: () => void): void;
```

Comment:
FYI - although I think this is a good rename, it's technically a breaking change since this is a public facing API 😢 

The good news is that it's fixed in the `v3.0.0-dev` branch via https://github.com/ionic-team/stencil/pull/3290